### PR TITLE
Awesome score sign

### LINKED ISSUE #203: Show awesome 'Score' sign on pacmacs-score command

### PR DESCRIPTION
**Pull request by rexim**
Close #203

### CODE REVIEW

**Comment by ForNeVeR** [comment]
Ok, I approve these.

**Comment by rexim** [comment]
Goat you mr @ForNeVeR. Appreciate it. :goat:

**Comment by rexim** [comment]
Linux ATT before and after the changes: `~7ms`

**Comment by rexim** [comment]
Windows ATT before and after the changes: `~10ms`

**Comment by rexim** [comment]
@ForNeVeR I've made some changes since your initial review. Could you please perform an additional review? Please merge the PR as the sign of approval. Thanks!

File: pacmacs-render.el [hunk 1]
```diff
@@ -62,19 +68,34 @@
         (pacmacs--render-anim anim))
     (pacmacs--render-empty-cell)))
 
-(defun pacmacs--render-track-board (track-board)
+(defun pacmacs--render-first-object-in-list (anim-objects)
```
Comment (ForNeVeR) [inline pacmacs-render.el]: This function seems too specific to me. Consider replacing it with lambda or with function composition (if dash.el has any).

Comment (rexim) [inline pacmacs-render.el]: Done!

**Comment by ForNeVeR** [comment]
Okay, I approve this yet again.

**Comment by rexim** [comment]
@ForNeVeR thank you very much! :)

**Comment by Newlifer** [comment]
I second this.

**Pull request merged**